src/client/runner/automation/automation.js

```diff
@@ -88,19 +89,19 @@ window[AUTOMATION_RUNNERS] = {
     dblclick: {
         playback: dblClickPlayback
     },
-    drag:     {
-        playback: dragPlayback
-    },
-    select:   {
+
+    DragAutomation: DragAutomation,
```

Comment:
We have inconsistency here. We haven't change `press` to `PressAutomation` [here](https://github.com/DevExpress/testcafe/pull/335/files#diff-38d8e03ca3f6059c75b7af48d907a48aR98)

So we should change it for `press` then. And let's change this to `window[AUTOMATIONS]` and change `AUTOMATIONS` const to something like `runner|automations`